docs/reference/modules/ROOT/pages/queries.adoc

```diff
@@ -3,27 +3,37 @@
 [#intro]
 == Introduction
 
-Crux is a document database that provides you with a comprehensive means of traversing and querying across all of your documents and data without any need to define a schema ahead of time.
-This is possible because Crux is "schemaless" and automatically indexes the top-level fields in all of your documents to support efficient ad-hoc joins and retrievals.
-With these capabilities you can quickly build queries that match directly against the relations in your data without worrying too much about the shape of your documents or how that shape might change in future.
+Crux is a schemaless document database that provides you with a comprehensive means of traversing and querying across all of your documents.
+Crux automatically indexes the top-level fields in all documents, supporting efficient ad-hoc joins and retrievals.
+Crux is both _immutable_ and _bitemporal_, which are both required characteristics for a safe schemaless database.
 
 Crux is also a graph database.
 The central characteristic of a graph database is that it can support arbitrary-depth graph queries (recursive traversals) very efficiently by default, without any need for schema-level optimisations.
-Crux gives you the ability to construct graph queries via a Datalog query language and uses graph-friendly indexes to provide a powerful set of querying capabilities.
-Additionally, when Crux's indexes are deployed directly alongside your application you are able to easily blend Datalog and code together to construct highly complex graph algorithms.
+Graph queries are possible through the Crux dialect of the Datalog query language.
```

Comment:
I don't prefer the phrasing of "the Crux dialect of the Datalog query language", at least without giving more background on what Datalog is (links, perhaps even snippets)